style.css

```diff
@@ -0,0 +1,111 @@
+.wrapper {
+	width: 80%;
+	margin: auto;
+	border: dashed 1px red;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6226470)